### DISCUSSION TITLE
architectural image advice

### DISCUSSION THREAD

**Discussion by mashb1t** [lllyasviel/Fooocus#2900] [Q&A] [answered]
Hello,
We need convert hand drawing sketch (architectural) to render or photorealism but do not change the building concept, only make different variation of materials and add some environment & lighting etc., but no matter how hard I try it is changing the building concepts, what should we do?
Usually in generation preview until step 10 it is very close to the main concept, but suddenly totally changes the design/environment
Original sketch:
![20240506_093407](https://github.com/lllyasviel/Fooocus/assets/161326842/8fbf465b-ef06-47a4-b42a-6bc1e0249f04)
Results with/without prompt, any setting such as control net/sketch ,mode etc.
 Results:
![01](https://github.com/lllyasviel/Fooocus/assets/161326842/fdd0e362-4222-4086-bd1d-abcf38f9f34c)
![02](https://github.com/lllyasviel/Fooocus/assets/161326842/abd28a01-40fb-4845-b28f-0a47a16eab02)

_Originally posted by @kad-studio in https://github.com/lllyasviel/Fooocus/discussions/557#discussioncomment-9351472_

**Comment by mashb1t** [comment]
@kad-studio it seems like you're using an old version of Fooocus missing multiple features and improvements. Please make sure you're using the latest version to guarantee reproducible results in high quality. To do so, please run the entry_with_update.py instead of launch.py and/or modify your run.bat to use this file.
Alternatively you can also run git pull manually in your Fooocus directory or download Fooocus again.

I've noticed that you also use the sdxl offset LoRA in strength 0.5. Please change this to 0.1 as a high value can have a negative impact here.

You can also try to switch the model to Juggernaut (the default in newer Fooocus versions) as an improved base model, as the SAI SDXL model + refiner is IMHO inferior of Juggernaut, but please try this yourself.

Hope these steps help in bumping the quality and in achieving your goal. Let's stay in touch in this discussion!

@tybiboune fyi

**Comment by kad-studio** [comment]
Hello @tybiboune
Thanks you and @mashb1t for your responds
Your first result in https://github.com/lllyasviel/Fooocus/discussions/557 was so close to what we are looking for, May I ask what exactly you did set and wire command?
I updated Fooocus Ai and here is my settings, what is wrong with mine is not ideal result like yours? 
![01](https://github.com/lllyasviel/Fooocus/assets/161326842/a8fb901b-b5a0-49c1-800a-117de8861737)
![02](https://github.com/lllyasviel/Fooocus/assets/161326842/30c8f038-1874-475e-94fb-75690344a4a3)
![03](https://github.com/lllyasviel/Fooocus/assets/161326842/87972632-e7e8-407a-8ea3-fd12bd8aa00c)
![04](https://github.com/lllyasviel/Fooocus/assets/161326842/888bc83a-faa7-4379-82f1-b0d1da213808)

Best regards

**Reply by tybiboune** [reply, marked as answer]
hiya! sorry, I've helped to certain limits, as I understand that you're probably doing this for professionnal (commercial :P ) exploitation. Fooocus / SD are free and open source, but proper use of the tools is a skill, I'm fine with sharing but in those conditions it seems a bit unfair ;) I'm available for any short mission though! ;P

**Reply by kad-studio** [reply]
Hello, Oh didn't know that, thanks :(

**Reply by kad-studio** [reply]
Only this 1 sample could give me a guide of total workflow if its possible do me a favor plz and show me

**Comment by Rogala** [comment]
Tried to do, but nothing good did not work, with Lightning does not have time to draw everything, with Speed on the contrary does not look like and checkpoints juggernautXL_v8Rundiffusion also did not give normal results. 

Only I cleaned up the original image a bit, I think the background interferes. And used realismEngineSDXL_v30VAE with Extreme Speed 

`{
"prompt": "A building with large glass windows two stories high, an archway with double doors in the center of the building, a rectangular fountain in front of the building, paths on the sides of the fountain, trees next to the paths, clear skies, mid afternoon",
"negative_prompt": "",
"prompt_expansion": "A building with large glass windows two stories high, an archway with double doors in the center of the building, a rectangular fountain in front of the building, paths on the sides of the fountain, trees next to the paths, clear skies, mid afternoon, warm colors, highly detailed, very vibrant, intricate, cinematic light, sharp focus, adventurous, atmosphere, magical",
"styles": "['Fooocus Sharp', 'Fooocus Enhance', 'Fooocus V2']",
"performance": "Extreme Speed",
"resolution": "(1600, 640)",
"guidance_scale": 1.0,
"sharpness": 0.0,
"adm_guidance": "(1.0, 1.0, 0.0)",
"base_model": "realismEngineSDXL_v30VAE.safetensors",
"refiner_model": "None",
"refiner_switch": 1.0,
"adaptive_cfg": 1.0,
"sampler": "lcm",
"scheduler": "lcm",
"seed": "2047989093416327970",
"lora_combined_1": "sd_xl_offset_example-lora_1.0.safetensors : 0.4",
"lora_combined_2": "sdxl_lcm_lora.safetensors : 1.0",
"metadata_scheme": false,
"version": "Fooocus v2.3.1"
}`
@
![11111](https://github.com/lllyasviel/Fooocus/assets/7437582/52b10148-cd5d-472b-8ed3-9b838c94c3d7)
![2222](https://github.com/lllyasviel/Fooocus/assets/7437582/2f750737-5c2f-4982-9f4d-faeddfa74a44)
![2024-05-12_183315](https://github.com/lllyasviel/Fooocus/assets/7437582/b88a5a6b-16bb-48eb-8b6d-3ae5e8966603)
